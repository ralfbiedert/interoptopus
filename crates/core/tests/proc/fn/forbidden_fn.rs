use interoptopus::ffi_function;

#[ffi_function]
fn new() {}

fn main() {}
