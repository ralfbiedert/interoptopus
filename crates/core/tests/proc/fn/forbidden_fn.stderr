error: Using the name 'new' can cause conflicts in generated code.
 --> tests/proc/fn/forbidden_fn.rs:4:4
  |
4 | fn new() {}
  |    ^^^
